src/common/__tests__/BaseAnnotator-test.ts

```diff
@@ -38,11 +38,19 @@ describe('BaseAnnotator', () => {
         locale: 'en-US',
         token: '1234567890',
     };
-    const getAnnotator = (options = {}): MockAnnotator => new MockAnnotator({ ...defaults, ...options });
-    let annotator = getAnnotator();
+    const annotatorFactory = {
+        annotator: new MockAnnotator({ ...defaults }),
+        getAnnotator(options = {}) {
+            this.annotator.destroy();
```

Comment:
Can we wrap this is in `if (this.annotator)` and avoid line 42?

Reply:
Throws a typescript error if I try that, saying that `this.annotator` Throws a typescript error if I try that, saying that property `annotator` doesn't exist

Reply:
I was thinking something like this. Not a big deal, either way.

```
let annotator: MockAnnotator;

const initAnnotator = (options = {}): void => {
    if (annotator) {
        annotator.destroy();
    }

    annotator = new MockAnnotator({ ...defaults, ...options });
};

beforeEach(() => {
    initAnnotator();
});
```